packages/vscode-extension/src/Logger.ts

```diff
@@ -1,6 +1,9 @@
-import { LogOutputChannel, window } from "vscode";
+import { Output } from "./common/OutputChannel";
+import { IDE } from "./project/ide";
 
-const outputChannel = window.createOutputChannel("Radon IDE", { log: true });
+const getIdeOutputChannel = () => {
+  return IDE.getInstanceIfExists()?.outputChannelRegistry.getOrCreateOutputChannel(Output.Ide);
```

Comment:
this isn't good as it will implicitly create IDE instance. While logs can be produces before we can instantiate the IDE. I think it'd be better to convert the OutputChannelRegistry to be accessible as singleton. The class is stateless, so it should be fine to make the exception in this case